packages/salesforcedx-apex-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -163,13 +164,63 @@ export class ScopeContainer extends VariableContainer {
   }
 }
 
+export class ApexHeapDump {
+  private readonly heapDumpId: string;
+  private readonly className: string;
+  private readonly namespace: string;
+  private readonly line: number;
+  private overlaySuccessResut:
+    | ApexExecutionOverlayResultCommandSuccess
+    | undefined;
+  public constructor(
+    heapDumpId: string,
+    className: string,
+    namespace: string,
+    line: number
+  ) {
+    this.heapDumpId = heapDumpId;
+    this.className = className;
+    this.namespace = namespace;
+    this.line = line;
+  }
+  public getHeapDumpId(): string {
+    return this.heapDumpId;
+  }
+  public getClassName(): string {
+    return this.className;
+  }
+  public getNamespace(): string {
+    return this.namespace;
+  }
+  public getLine(): number {
+    return this.line;
+  }
+  public getOverlaySuccessResult():
+    | ApexExecutionOverlayResultCommandSuccess
+    | undefined {
+    return this.overlaySuccessResut;
+  }
+  public setOverlaySuccessResult(
+    overlaySuccessResult: ApexExecutionOverlayResultCommandSuccess
+  ): void {
+    this.overlaySuccessResut = overlaySuccessResult;
+  }
+  /* tslint:disable */
+  public toString = (): string => {
```

Comment:
What is the reason for defining `toString` this way instead of `public toString():string {...}`? Won't that avoid the issue with the semicolon?